{"server": "GraphQL-DB", "asctime": "2024-03-06 16:29:36,979", "level": "ERROR", "name": "root", "message": "(<class 'httpx.ConnectError'>, ConnectError('All connection attempts failed'), <traceback object at 0x0000013389ABF2C0>) Query: send_request, REST From: 34"}
{"server": "GraphQL-DB", "asctime": "2024-03-07 12:26:35,530", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:10:20,913", "level": "INFO", "name": "httpx", "message": "HTTP Request: POST http://127.0.0.1:5001/login/ \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:10:30,439", "level": "INFO", "name": "httpx", "message": "HTTP Request: POST http://127.0.0.1:5001/login/ \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:11:06,883", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:11:15,750", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:12:07,162", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:12:09,285", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 13:30:31,816", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 14:34:36,939", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 14:40:08,461", "level": "ERROR", "name": "admin.graphql", "message": "Cannot query field 'getUserx' on type 'Query'. Did you mean 'getUser', 'getOrder', or 'getUrls'?\n\nGraphQL request:3:17\n2 |             query GetUser ($page: Int, $limit: Int) {\n3 |                 getUserx(page: $page, limit: $limit) {\n  |                 ^\n4 |                     message", "exc_info": "graphql.error.graphql_error.GraphQLError: Cannot query field 'getUserx' on type 'Query'. Did you mean 'getUser', 'getOrder', or 'getUrls'?\n\nGraphQL request:3:17\n2 |             query GetUser ($page: Int, $limit: Int) {\n3 |                 getUserx(page: $page, limit: $limit) {\n  |                 ^\n4 |                     message"}
{"server": "GraphQL-DB", "asctime": "2024-03-07 15:03:54,152", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-07 15:04:17,930", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:13:38,632", "level": "ERROR", "name": "root", "message": "(<class 'redis.exceptions.ConnectionError'>, ConnectionError('Error 10061 connecting to localhost:6379. No connection could be made because the target machine actively refused it.'), <traceback object at 0x000001D498A8D740>) Query: get, redis From: 39"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:13:38,703", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('DataLoader must be constructed with a function which accepts Iterable<key> and returns Future<Iterable<value>>, but the function did not return a Future of a Iterable: False.'), <traceback object at 0x000001D498A94A00>) Query: resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:23:04,910", "level": "ERROR", "name": "root", "message": "(<class 'redis.exceptions.DataError'>, DataError(TypeError('A tuple item must be str, int, float or bytes.')), <traceback object at 0x000001D498BB3E40>) Query: get, redis From: 39"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:23:04,911", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('DataLoader must be constructed with a function which accepts Iterable<key> and returns Future<Iterable<value>>, but the function did not return a Future of a Iterable: False.'), <traceback object at 0x000001D498B70B80>) Query: resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:26:43,950", "level": "ERROR", "name": "root", "message": "(<class 'redis.exceptions.DataError'>, DataError(TypeError('A tuple item must be str, int, float or bytes.')), <traceback object at 0x000001FAA85C4640>) Query: get, redis From: 39"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:26:43,951", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('DataLoader must be constructed with a function which accepts Iterable<key> and returns Future<Iterable<value>>, but the function did not return a Future of a Iterable: False.'), <traceback object at 0x000001FAA85FC840>) Query: resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:28:16,504", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('DataLoader must be constructed with a function which accepts Iterable<key> and returns Future<Iterable<value>>, but the function did not return a Future of a Iterable with the same length as the Iterable of keys.\\n\\nKeys:\\n[1]\\n\\nValues:\\n[]'), <traceback object at 0x000001F16D443AC0>) Query: resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:28:42,273", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('DataLoader must be constructed with a function which accepts Iterable<key> and returns Future<Iterable<value>>, but the function did not return a Future of a Iterable: False.'), <traceback object at 0x0000016BD30C4840>) Query: resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:31:13,731", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('DataLoader must be constructed with a function which accepts Iterable<key> and returns Future<Iterable<value>>, but the function did not return a Future of a Iterable with the same length as the Iterable of keys.\\n\\nKeys:\\n[1]\\n\\nValues:\\n[]'), <traceback object at 0x000001D5EF944080>) Query: resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:31:56,324", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:39:07,453", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:39:08,673", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:39:09,618", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:39:10,571", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:40:42,899", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:43:16,114", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:43:58,337", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:49:16,891", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:49:36,376", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 12:51:19,881", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 13:29:24,360", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"module 'jwt' has no attribute 'decode'\"), <traceback object at 0x0000022F73FDA580>) Query: Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 13:29:24,397", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:18:34,190", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('key must be an instance of a class implements jwt.AbstractJWKBase'), <traceback object at 0x000002447EDA4400>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:18:34,231", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:22:22,574", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:22:22,574", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError('key must be an instance of a class implements jwt.AbstractJWKBase'), <traceback object at 0x0000014AB77A4600>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:22:22,602", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:39:28,952", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:39:28,953", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x00000257C08B6C00>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:39:28,982", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=10 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:25,544", "level": "INFO", "name": "root", "message": " : LoginUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:25,544", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x00000261C4CD9540>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:25,546", "level": "ERROR", "name": "admin.graphql", "message": "Expected value of type 'String!', found null.\n\nGraphQL request:2:27\n1 | query LoginUser {\n2 |     loginUser(login_user: null, login_password: null) {\n  |                           ^\n3 |         message", "exc_info": "graphql.error.graphql_error.GraphQLError: Expected value of type 'String!', found null.\n\nGraphQL request:2:27\n1 | query LoginUser {\n2 |     loginUser(login_user: null, login_password: null) {\n  |                           ^\n3 |         message"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:25,547", "level": "ERROR", "name": "admin.graphql", "message": "Expected value of type 'String!', found null.\n\nGraphQL request:2:49\n1 | query LoginUser {\n2 |     loginUser(login_user: null, login_password: null) {\n  |                                                 ^\n3 |         message", "exc_info": "graphql.error.graphql_error.GraphQLError: Expected value of type 'String!', found null.\n\nGraphQL request:2:49\n1 | query LoginUser {\n2 |     loginUser(login_user: null, login_password: null) {\n  |                                                 ^\n3 |         message"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:41,326", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:41,327", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x00000261C4CE5780>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:40:41,356", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:46:06,150", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:46:06,151", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x000001D8CE5F8C80>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:46:06,181", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:51:24,834", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:51:24,835", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x000001D8CE6A55C0>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:51:24,875", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:52:13,472", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:52:13,472", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x000001D8CE6BBB80>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:52:13,506", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:53:35,485", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:53:35,485", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x00000250FF4B58C0>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:53:35,524", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:54:42,504", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:54:42,504", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x00000250FF532680>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:54:42,528", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:55:36,405", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:55:36,405", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x00000209FEEADB40>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:55:36,441", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:56:18,489", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:56:18,489", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x000001A11CC40A40>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:56:18,515", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:56:58,077", "level": "INFO", "name": "root", "message": " : No_name - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:56:58,077", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.InvalidSignatureError'>, InvalidSignatureError('Signature verification failed'), <traceback object at 0x000001A11CCBE8C0>) : Token decode From: 23"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:57:04,812", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 14:57:04,834", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:05:08,293", "level": "INFO", "name": "root", "message": " : CreateUsers - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:15:50,703", "level": "INFO", "name": "root", "message": " : CreateUsers - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:15:50,706", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError(\"'coroutine' object is not subscriptable\"), <traceback object at 0x0000020D48CF2F00>) : resolve_users, mutation From: 50"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:16:15,961", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:16:15,963", "level": "ERROR", "name": "root", "message": "(<class 'KeyError'>, KeyError('user_loader'), <traceback object at 0x0000020D497B1200>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:18:41,905", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:18:41,907", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'coroutine' object has no attribute 'load'\"), <traceback object at 0x0000021E020A9900>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:21:50,909", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:21:50,911", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'coroutine' object has no attribute 'load'\"), <traceback object at 0x00000236C649B9C0>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:23:23,347", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:23:23,349", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'coroutine' object has no attribute 'load'\"), <traceback object at 0x000002A040219900>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:24:00,588", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:24:00,590", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'coroutine' object has no attribute 'load'\"), <traceback object at 0x000001C359F1E7C0>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:24:39,901", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:24:39,903", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'str' object has no attribute 'load'\"), <traceback object at 0x000001F5D8315500>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:25:07,286", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:25:07,293", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'str' object has no attribute 'load'\"), <traceback object at 0x000002C4BA925C40>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:29:40,430", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:29:40,437", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'str' object has no attribute 'load'\"), <traceback object at 0x0000023A82C21B80>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:29:48,385", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:29:48,388", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'str' object has no attribute 'load'\"), <traceback object at 0x0000023A82C18380>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:31:00,508", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:31:00,515", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'str' object has no attribute 'load'\"), <traceback object at 0x000002438FEEF240>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:31:03,707", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 15:31:03,710", "level": "ERROR", "name": "root", "message": "(<class 'AttributeError'>, AttributeError(\"'str' object has no attribute 'load'\"), <traceback object at 0x000002438FEE6D40>) : resolve_users, query From: 74"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 16:09:39,737", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 16:09:39,743", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError(\"object str can't be used in 'await' expression\"), <traceback object at 0x0000016BBE5E7FC0>) : resolve_users, query From: 75"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 16:14:09,461", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 16:14:09,491", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 16:14:26,171", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 16:14:26,193", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:09:40,733", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:10:38,477", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:10:38,484", "level": "ERROR", "name": "admin.graphql", "message": "must be str, not GraphQLResolveInfo\n\nGraphQL request:8:9\n7 |         previous\n8 |         count\n  |         ^\n9 |     }", "exc_info": "Traceback (most recent call last):\n  File \"D:\\Backend\\auditando\\graphql\\.auditando_ariadne\\lib\\site-packages\\graphql\\execution\\execute.py\", line 521, in execute_field\n    result = resolve_fn(source, info, **args)\n  File \"D:\\Backend\\auditando\\graphql\\.auditando_ariadne\\lib\\site-packages\\graphql\\execution\\execute.py\", line 1250, in default_field_resolver\n    return value(info, **args)\nTypeError: must be str, not GraphQLResolveInfo\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"D:\\Backend\\auditando\\graphql\\.auditando_ariadne\\lib\\site-packages\\graphql\\execution\\execute.py\", line 521, in execute_field\n    result = resolve_fn(source, info, **args)\n  File \"D:\\Backend\\auditando\\graphql\\.auditando_ariadne\\lib\\site-packages\\graphql\\execution\\execute.py\", line 1250, in default_field_resolver\n    return value(info, **args)\ngraphql.error.graphql_error.GraphQLError: must be str, not GraphQLResolveInfo\n\nGraphQL request:8:9\n7 |         previous\n8 |         count\n  |         ^\n9 |     }"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:12:15,871", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:12:15,907", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:13:40,411", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:10,998", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:11,032", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:23,190", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:33,069", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:33,091", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 200 OK\""}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:36,637", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:37,816", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:38,868", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:39,889", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-09 17:14:40,810", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:24:59,100", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x0000015520773A40>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:24:59,100", "level": "INFO", "name": "root", "message": " : No_name - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:25:02,591", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x00000155207BCDC0>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:25:02,591", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:25:02,665", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:30:30,318", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x00000155207C7DC0>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:30:30,318", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:30:30,339", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:31:59,745", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:31:59,746", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x000001FEE2CEB640>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:31:59,774", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:42,850", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:42,850", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x0000027A81063C40>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:42,879", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:45,783", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:45,784", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x0000027A81119C00>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:45,802", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:58,617", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:58,617", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x00000158C302AD80>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:32:58,646", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:36:06,742", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:36:06,742", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x000001FDC97D3AC0>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:36:06,771", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:36:08,632", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:36:08,632", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x000001FDCA0A40C0>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:36:08,654", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:42:12,260", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:42:41,985", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:42:48,524", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:42:50,092", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:42:50,924", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:43:33,546", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:43:35,311", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:43:36,311", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:46:54,072", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:46:54,073", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x000002289E3E49C0>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:46:54,075", "level": "ERROR", "name": "root", "message": "(<class 'NameError'>, NameError(\"name 'a' is not defined\"), <traceback object at 0x000002289DB7CE80>) : resolve_users, query From: 78"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:47:23,046", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:47:23,047", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x000002E3269CAD00>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:47:23,050", "level": "ERROR", "name": "ariadne", "message": "name 'a' is not defined\n\nGraphQL request:2:5\n1 | query GetUser {\n2 |     getUser {\n  |     ^\n3 |         message", "exc_info": "Traceback (most recent call last):\n  File \"D:\\Backend\\auditando\\graphql\\.auditando_ariadne\\lib\\site-packages\\graphql\\execution\\execute.py\", line 528, in await_result\n    return_type, field_nodes, info, path, await result\n  File \"D:\\Backend\\auditando\\graphql\\api\\graphql\\query\\__init__.py\", line 77, in resolve_getUser\n    a\nNameError: name 'a' is not defined\n\nThe above exception was the direct cause of the following exception:\n\nTraceback (most recent call last):\n  File \"D:\\Backend\\auditando\\graphql\\.auditando_ariadne\\lib\\site-packages\\graphql\\execution\\execute.py\", line 528, in await_result\n    return_type, field_nodes, info, path, await result\n  File \"D:\\Backend\\auditando\\graphql\\api\\graphql\\query\\__init__.py\", line 77, in resolve_getUser\n    a\ngraphql.error.graphql_error.GraphQLError: name 'a' is not defined\n\nGraphQL request:2:5\n1 | query GetUser {\n2 |     getUser {\n  |     ^\n3 |         message"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:53:51,537", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:53:51,538", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x0000022ED7B16980>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 16:53:51,566", "level": "INFO", "name": "httpx", "message": "HTTP Request: GET http://127.0.0.1:5001/users/?page=1&limit=5 \"HTTP/1.1 401 Unauthorized\""}
{"server": "GraphQL-DB", "asctime": "2024-03-10 23:26:05,050", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 23:26:10,364", "level": "INFO", "name": "root", "message": " : GetUser - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 23:26:10,364", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x000001F7CF233640>) : Token decode From: 22"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 23:26:14,433", "level": "ERROR", "name": "root", "message": "(<class 'redis.exceptions.ConnectionError'>, ConnectionError('Error 10061 connecting to localhost:6379. No connection could be made because the target machine actively refused it.'), <traceback object at 0x000001F7CF233D80>) : get, redis From: 39"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 23:26:16,468", "level": "ERROR", "name": "root", "message": "(<class 'httpx.ConnectError'>, ConnectError('All connection attempts failed'), <traceback object at 0x000001F7CF28BB00>) : send_request, REST From: 34"}
{"server": "GraphQL-DB", "asctime": "2024-03-10 23:33:44,467", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 08:53:48,752", "level": "INFO", "name": "root", "message": " : IntrospectionQuery - No_variables From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 09:29:05,077", "level": "ERROR", "name": "root", "message": "(<class 'starlette.exceptions.HTTPException'>, HTTPException(status_code=405, detail='Method Not Allowed'), <traceback object at 0x000001DD563D7C80>) : send_request, REST From: 44"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 09:29:05,077", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError(\"'dict' object is not callable\"), <traceback object at 0x000001DD563907C0>) : send_request, REST From: 164"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 09:52:56,242", "level": "ERROR", "name": "root", "message": "(<class 'starlette.exceptions.HTTPException'>, HTTPException(status_code=405, detail='Method Not Allowed'), <traceback object at 0x0000020546D052C0>) : Internal Error, exception_handlers From: 44"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 09:52:56,243", "level": "ERROR", "name": "root", "message": "(<class 'TypeError'>, TypeError(\"'dict' object is not callable\"), <traceback object at 0x0000020546D053C0>) : Internal Error, exception_handlers From: 164"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 11:28:27,519", "level": "ERROR", "name": "root", "message": "Exception handlers : Method Not Allowed From: 405"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 11:38:08,468", "level": "ERROR", "name": "root", "message": "Exception handlers : Method Not Allowed From: 405"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 11:38:23,966", "level": "ERROR", "name": "root", "message": "Exception handlers : Method Not Allowed From: 405"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 11:38:36,529", "level": "ERROR", "name": "root", "message": " Exception_handlers : Method Not Allowed - 405 From: http://127.0.0.1:5000/"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 13:22:19,690", "level": "ERROR", "name": "root", "message": " Exception_handlers : Method Not Allowed - 405 From: http://127.0.0.1:5000/"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 13:22:22,565", "level": "INFO", "name": "root", "message": " : No_name - {} From: Graphql"}
{"server": "GraphQL-DB", "asctime": "2024-03-11 13:22:22,565", "level": "ERROR", "name": "root", "message": "(<class 'jwt.exceptions.ExpiredSignatureError'>, ExpiredSignatureError('Signature has expired'), <traceback object at 0x0000028E90436240>) : Token decode From: 26"}
